src/main/java/com/github/freeclimbapi/utils/SignatureInformation.java

```diff
@@ -0,0 +1,60 @@
+package com.github.freeclimbapi.utils;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Date;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import org.apache.commons.codec.binary.Hex;
+
+public class SignatureInformation {
+    public int requestTimestamp;
+    public List<String> signatures;
+
+    public SignatureInformation(String requestHeader) {
+        List<String> constructorSignatures = new ArrayList<String>();
```

Comment:
rather than reassigning, lets just assign `signatures = new ArrayList<String>()`

Reply:
will change in latest commit